<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DesktopFileCreator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>-5</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>DesktopFileCreator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameField">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>45</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="execField">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>85</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Executable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iconField">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>125</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Icon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commentField">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>165</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="terminalCheckBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>210</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run in terminal?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execBrowseButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>85</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iconBrowseButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>125</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Categories">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>245</y>
      <width>350</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>Categories:</string>
    </property>
    <widget class="QCheckBox" name="categoryDevelopment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Development</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categoryGames">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>30</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Games</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categoryGraphics">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>30</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Graphics</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categoryInternet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Internet</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categoryOffice">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>60</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Office</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categoryMultimedia">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>60</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Multimedia</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categorySettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categorySystem">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>90</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>System</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="categoryUtility">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>90</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Utility</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="installLocationGroup">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>385</y>
      <width>350</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Install location:</string>
    </property>
    <widget class="QRadioButton" name="userRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>300</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>User-only ~/.local/share/applications</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="systemRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>300</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>System-wide /usr/share/applications</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="customRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="customPathField">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>115</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="customPathBrowseButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>115</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="createDesktopFileButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>550</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Install File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>595</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;https://beeralator.com&quot;&gt;Beeralator.com&lt;/a&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextBrowserInteraction|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
